src/components/SymbolModal.js

```diff
@@ -162,27 +167,33 @@ class SymbolModal extends Component {
       symbols
     } = this.props;
 
-    if (!query || !selectedSource) {
+    if (!selectedSource) {
       return;
     }
 
-    const symbolSearchResults = filter(symbols[symbolType], query, {
+    let symbolSearchResults = symbols[symbolType];
+
+    if (query == "") {
+      this.setState({ results: symbolSearchResults });
```

Comment:
thinking about switching to local state because the state is only specific to the modal.